src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1001,6 +1002,9 @@ public Module getModule() {
     private transient Object classData; // Set by VM
     private transient Object[] signers; // Read by VM, mutable
 
+    @Stable
```

Comment:
The `modifiers` field doesn’t need to be `@Stable`:
```suggestion
```

Reply:
I now don't know whether we want @Stable here or not.